lib/cds_importer/entity_mapper.rb

```diff
@@ -57,8 +59,14 @@ def save_record(record)
       nil
     end
 
+    def remove_excluded_geographical_areas!
+      return if xml_node['sid'].blank?
+
+      MeasureExcludedGeographicalArea.where(measure_sid: xml_node['sid']).delete
```

Comment:
So I think that you can do

```suggestion
      MeasureExcludedGeographicalArea.operation_klass.where(measure_sid: xml_node['sid']).delete
```

And not need the change to the overridden delete method in the oplog plugin.